frontend/src/metabase/dashboard/hoc/DashboardControls.jsx

```diff
@@ -127,9 +127,14 @@ export default ComposedComponent =>
         }
       };
 
+      // Preserve existing behavior, while keeping state in a new `theme` key
```

Comment:
Preserve the API, because there are like 5-6 components that still need `isNightMode`, and I don't want to do that now.